run-export.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-DOCS=markdown/en
+DOCS=markdown/3.0/en
```

Comment:
why do we not have to export 1.2? (just aksing, I have no idea what this file does)

Reply:
It's a temporary thing, we have to implement multi-version support in our docs export tool and on the FACT-Finder side.